### PR TITLE
Uncomment/unbreak FILE=<<EOF tests

### PR DESCRIPTION
**Pull request by kazarmy**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [X] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

This pr unbreaks the `FILE=<<EOF` tests so that they are not forgotten and to test them with the new r2r.c.

**Test plan**

Travis and AppVeyor are green.

**Closing issues**

...

### CODE REVIEW

File: test/new/db/cmd/cmd_open [hunk 1]
```diff
@@ -260,30 +260,29 @@ isq~?
...
-#2 5
-#
-#fd       4
-#fd       5
-#fd       3
-#EOF
-#RUN
+NAME=ob 0 fix
+FILE=<<EOF
+-
```
Comment (radare) [inline test/new/db/cmd/cmd_open:265]: this doesnt looks good at all

Comment (kazarmy) [reply test/new/db/cmd/cmd_open]: ok fixed 59a7711574e1e434d2e380d9d52f5f0b1394d244

File: test/new/db/io/many [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
-#-
-#EOF
-#CMDS=o~?
-#EXPECT=<<EOF
-#3
-#EOF
-#RUN
+NAME=multiple files opened
+FILE=<<EOF
+-
```
Comment (radare) [inline test/new/db/io/many:3]: and this?

File: test/new/db/tools/r2r [hunk 3]
```diff
@@ -44,23 +43,22 @@ EXPECT_ERR=<<EOF
...
-#2 5 x86-32 ba:0x08048000 sz:13073 ../bins/elf/_Exit (42)
-#3 6 x86-64 ba:0x00000000 sz:512 malloc://512
-#4 7 x86-64 ba:0x00000000 sz:512 malloc://512
-#EOF
-#EXPECT_ERR=<<EOF
-#EOF
-#RUN
+NAME=bin with space in filename (multiple files)
+FILE=<<EOF
+-
```
Comment (radare) [inline test/new/db/tools/r2r:48]: or this

File: test/new/db/io/many (hunk 2 above)
Comment (kazarmy) [reply test/new/db/io/many]: ok f9b988a07b1d8408d1ff1f837d7fedd726d41a5e

File: test/new/db/tools/r2r (hunk 3 above)
Comment (kazarmy) [reply test/new/db/tools/r2r]: fixed this by removing it since it's unnecessary 77a1eb79ee041df4ed646efc7657bcfa6125b466

**Pull request merged**